samples/word/99-preview-apis/manage-fields.yaml

```diff
@@ -0,0 +1,120 @@
+id: word-manage-fields
+name: Get fields
+description: This sample shows how to get existing fields.
+host: WORD
+api_set:
+    WordApi: '1.4'
+script:
+    content: |
+        $("#run").click(() => tryCatch(run));
+        $("#setup").click(() => tryCatch(setup));
+
+        async function run() {
+          await Word.run(async (context) => {
+            let field = context.document.body.fields.getFirstOrNullObject();
+            field.load();
+
+            await context.sync();
+
+            console.log("Code: " + field.code);
+            console.log("Result: " + field.result);
+
+            let parentBody = field.parentBody;
+            parentBody.load("text");
+            await context.sync();
+
+            console.log("Text of parentBody: " + JSON.stringify(parentBody.text));
+          });
+        }
+
+        async function setup() {
+          await Word.run(async (context) => {
+            context.document.body.clear();
+            context.document.body.insertParagraph("One more paragraph. ", "Start");
+            context.document.body.insertParagraph("Inserting another paragraph. ", "Start");
+            context.document.body.insertParagraph(
```

Comment:
Nit: since you're using `context.document.body` so much, consider saving the `Body` object as a variable to improve readability.

Reply:
Will add this to the backlog since this is used all over the place.